### PR TITLE
Added Entities workflow image on Datasets page

### PR DESCRIPTION
**Pull request by sadiqkhoja**
## Change:

- Added Entities workflow image on Datasets page 

https://user-images.githubusercontent.com/447837/203635093-e396812d-57f7-4ef8-9011-475fe5cf4587.mp4

### CODE REVIEW

File: src/components/dataset/introduction.vue [hunk 1]
```diff
@@ -51,8 +56,43 @@ export default {
...
+}
+
+@media (min-width: $screen-md-min) {
+  #dataset-introduction {
+    &.panel-dialog {
+      max-width: 1045px;
+    }
+
+    .row {
+      display: flex;
```
Comment (matthew-white) [inline src/components/dataset/introduction.vue:89]: Why is this needed? Isn't flexbox more of an alternative to the Bootstrap grid? (I actually like flexbox a lot and often reach for it over Boostrap.)

File: src/components/dataset/introduction.vue [hunk 2]
```diff
@@ -12,30 +12,35 @@ except according to the terms contained in the LICENSE file.
...
+          <i18n-t keypath="getStarted.full">
+            <template #clickHere>
+              <!-- TODO. Specify the `to` prop. -->
+              <doc-link>{{ $t('getStarted.clickHere') }}</doc-link>
+            </template>
+          </i18n-t>
+        </p>
+      </div>
+      <div class="col-md-9 image-wrapper">
+        <img src="../../../public/images/entities-intro.svg" alt="entities-workflow">
```
Comment (matthew-white) [inline src/components/dataset/introduction.vue:42]: Do we need `alt`? If we're going to specify `alt`, should we make it an i18n message?

**Review by matthew-white — APPROVED** [review]

File: src/components/dataset/introduction.vue (hunk 1 above)
Comment (sadiqkhoja) [reply src/components/dataset/introduction.vue:89]: I used it to vertically center the image on some screen widths

**Comment by matthew-white** [comment]
I'm going to go ahead and merge so we can get this to QA.

File: src/components/dataset/introduction.vue (hunk 2 above)
Comment (matthew-white) [reply src/components/dataset/introduction.vue:42]: Done in #660.
